app/controllers/api/v1/stats/stats_controller.rb

```diff
@@ -1,4 +1,5 @@
-class Api::StatsController < Api::BaseController
+class Api::V1::Stats::StatsController < Api::BaseController
```

Comment:
IMO the controller could be better named.

Perhaps EngagementController? @nvonpentz do you have any thoughts?

Reply:
I think stats fits since this info will be sent to the stats service and includes information that may not be related to engagement 

maybe DataController?

Reply:
Any consensus on this? It might make more sense to leave it 'as is' right now, wait for more stats to be built, then see if any patterns emerge.